backend/accounts_test.go

```diff
@@ -179,3 +182,86 @@ func TestNextAccountNumber(t *testing.T) {
 	_, err = nextAccountNumber(coinpkg.CodeTBTC, ks(fingerprint2, true), accountsConfig)
 	require.Equal(t, ErrAccountLimitReached, errp.Cause(err))
 }
+
+func newBackend(t *testing.T, testing, regtest bool) *Backend {
+	t.Helper()
+	b, err := NewBackend(
+		arguments.NewArguments(
+			test.TstTempDir("appfolder"),
+			testing, regtest,
+			false, false,
```

Comment:
It would be so much nicer if arguments were just a struct: I'll have to go and look up what `NewArguments` args are now...